### PR TITLE
displayMaxNumberOfNodes

### PR DESCRIPTION
**Pull request by tobwiens**
Add Alive nodes limit row

A alive node limit row is added. It shows the corresponding maxNumberOfNodes. If the alive nodes are not limited, the description is changed to alive nodes unlimited and the corresponding maxNumberOfNodes is displayed next to it, which is a negative number.

### CODE REVIEW

**Comment by tobwiens** [comment]
Limited alive nodes:



Unlimited alive nodes:

**Comment by tobwiens** [comment]
Here is the new PR for the changes. 

@lpellegr + I changed, the scheduling with constant 0 to fetch the data immeadiately after login.
+ I ran the auto formatting for it

**Comment by tobwiens** [comment]
@ow2-proactive/developers one other review please

**Comment by tobwiens** [comment]
@ow2-proactive/developers one review please.

**Comment by tobwiens** [comment]
@ow2-proactive/developers one review please.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java [hunk 1]
```diff
@@ -325,6 +326,24 @@ public void run() {
...
         this.statsUpdater.scheduleRepeating(RMConfig.get().getStatisticsRefreshTime());
+        }
+
+
+    private void fetchNodesLimit() {
+        this.rm.getState(LoginModel.getInstance().getSessionId(), new AsyncCallback<String>() {
+            public void onSuccess(String result) {
+                // Parse json response to extract the current node limit
+                JSONObject rmState = JSONParser.parseStrict(result).isObject();
+                JSONValue maxNumberOfNodes = rmState.get("maxNumberOfNodes");
```
Comment (marcocast) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java]: Should we check if the rmState json is correct? Something like 

```
if (rmState == null){
   LogModel.getInstance().logMessage("Failed to parse json rmState: " + rmstate);
}else{
   JSONValue maxNumberOfNodes = rmState.get("maxNumberOfNodes");
   if (maxNumberOfNodes != null && maxNumberOfNodes.isNumber() != null) {
            model.setMaxNumberOfNodes(Long.parseLong(maxNumberOfNodes.isNumber().toString()));
   }
}
```

Comment (tobwiens) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java]: Yes you are right.

**Pull request merged**